src/EventDispatcher/Subscriber/DoctrineProxySubscriber.php

```diff
@@ -53,7 +53,7 @@ public function onPreSerialize(PreSerializeEvent $event): void
         }
 
         if (($this->skipVirtualTypeInit && $virtualType) ||
-            (!$object instanceof Proxy && !$object instanceof ORMProxy)
+            (!$object instanceof Proxy && !$object instanceof LazyLoadingInterface)
```

Comment:
is `ORMProxy` still considered? does this work also with `ocramius/proxy-manager: 1.x`

Reply:
For `ORMProxy` I found that it implements `Proxy` interface and think that it could be dropped at all because if `!$object instanceof Proxy` then it is obviously `!$object instanceof ORMProxy`. So I replaced it with `LazyLoadingInterface`

For `ocramius/proxy-manager` you are right. It should work with 1.x and even with 0.1.0 version (very first). Should I lower version in composer.json?

Reply:
> For ORMProxy I found that it implements Proxy interface and think that it could be dropped at all because if !$object instanceof Proxy then it is obviously !$object instanceof ORMProxy. So I replaced it with LazyLoadingInterface 

I remember that, just if i'm not wrong wasn't so for all proxy-mamager versions

Reply:
> I remember that, just if i'm not wrong wasn't so for all proxy-mamager versions

Yes, proxy-manager has LaLoIn in all version, but I'm wondering how to reflect this in composer.json?
`"ocramius/proxy-manager": "*"` or `"ocramius/proxy-manager": "^0.1|^1.0|^2.0"`?

For ORMProxy, I have double checked with `--prefer-lowest` and `Doctrine\ORM\Proxy\Proxy` is instance of `Doctrine\Common\Proxy\Proxy`. Do you think it is still worth checking?

Reply:
I think if the tests are green with https://github.com/schmittjoh/serializer/pull/1139#discussion_r346219582, then we are good to go